src/styles/button.css

```diff
@@ -0,0 +1,36 @@
+.product-card__buy-button {
+  display: flex;
+  position: absolute;
+  left: 16px;
+  right: 16px;
+  top: 348px;
+  bottom: 16px;
+  width: 168px;
+  height: 40px;
+  justify-content: center;
+  align-items: center;
+  border-radius: 5px;
+  background: #00acdc;
+}
```

Comment:
If you write styles for element of the block - do it in the file with block styles.